index.html

```diff
@@ -16,21 +16,6 @@
     <meta property="og:url" content="http://repometric.com">
     <meta property="og:image" content="http://repometric.com/images/share-cover.jpg">
     
-    <!-- favicon -->
```

Comment:
Why links removed from HTML?

Reply:
The reason is to reduce duplication as the images are already described in manifest.json